custom/languages/en_UK.json

```
@@ -674,6 +675,7 @@
     "admin/view_site": "View Site",
     "admin/viewing_email_error": "Viewing error",
     "admin/viewing_integrations_for_x": "Viewing user integrations for {{user}}",
+    "admin/viewing_sessions_for_x": "Viewing user sessions for {{user}}",
```

```suggestion
    "admin/viewing_sessions_for_x": "Viewing login sessions for {{user}}",
```

Dunno, might read nicer